caffe2/ideep/utils/ideep_context.h

```diff
@@ -8,7 +8,7 @@
 
 namespace caffe2 {
 
-BaseStaticContext* GetIDEEPStaticContext();
+BaseStaticContext* CAFFE2_EXPORT GetIDEEPStaticContext();
```

Comment:
Actually, we have this error: 
```
caffe2/caffe2/ideep/utils/ideep_context.h:11:56: error: ‘__visibility__’ attribute ignored on non-class types [-Werror=attributes]
 BaseStaticContext* CAFFE2_EXPORT GetIDEEPStaticContext();
                                                        ^
```

Reply:
You probably need to change it to 
```
C10_EXPORT BaseStaticContext* GetIDEEPStaticContext();
```

BTW, due to recent changes, we need to replace `CAFFE2_EXPORT` to `C10_EXPORT`. 

And what is the reason to update ideep submodule again?